### PR TITLE
Defaults to base.py (OPS-4037)

### PR DESCRIPTION
**Pull request by nadeemshahzad**

### CODE REVIEW

**Comment by nadeemshahzad** [comment]
Tests are failing after I  set ```SAILTHRU_ENABLE: False``` as it is disabled in ecom-worker default configurations.

File: ecommerce_worker/configuration/base.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # CELERY
 # Default broker URL. See http://celery.readthedocs.org/en/latest/configuration.html#broker-url.
-BROKER_URL = None
+BROKER_URL = 'amqp://celery:celery@127.0.0.1:5672'
 
 # Disable connection pooling. Connections may be severed by load balancers.
 # This forces the application to connect explicitly to the broker each time
```
Comment (coryleeio) [inline ecommerce_worker/configuration/base.py:6]: Please ensure these values get included as well:




You need to include all these values as well:
https://github.com/edx/configuration/blob/master/playbooks/roles/edx_django_service/defaults/main.yml#L182-L222

as they get merged into the edx_django_service_config here:
https://github.com/edx/configuration/blob/master/playbooks/roles/edx_django_service/defaults/main.yml#L226

**Review by coryleeio — DISMISSED** [review]

File: ecommerce_worker/configuration/base.py (hunk 1 above)
Comment (nadeemshahzad) [reply ecommerce_worker/configuration/base.py:6]: @cory @adzuci Do we need these variables to add for IDAs (ecomworker and insights) for which ```edx_django_service``` is not loaded and executed as a dependency rather it include ```edx_service``` as a listed dependency in meta?
https://github.com/edx/configuration/blob/283e0046391456f604482145921d4b07d510f5c4/playbooks/roles/ecomworker/meta/main.yml#L16

**Comment by adzuci** [comment]
Though this fixes tests, it looked like SAILTHRU was set to false in configuration before, does enabling it pose any issue and do we still want to add the django_service values?

**Comment by nadeemshahzad** [comment]
This will not create any issue as I am enabling this in ```test.py``` and default it is going false to ```base.py``` from configuration. We are importing all settings from ```base.py``` to ```test.py``` and this setting was already true in ```test.py``` before de-dupe variables from configuration defaults. 
We don't need to include any variables from edx_service_config as it is empty dict ( https://github.com/edx/configuration/blob/283e0046391456f604482145921d4b07d510f5c4/playbooks/roles/edx_service/defaults/main.yml#L24)

**Review by adzuci — APPROVED** [review]
Per comments in https://openedx.atlassian.net/browse/OPS-4056 :shipit:

**Pull request merged**